samples/sensor/i3g4250d/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2021, Jonathan Hahn
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.13.1)
```

```suggestion
cmake_minimum_required(VERSION 3.20.0)
```

minimum CMake is now 3.20.0 on Zephyr `main`